maintenance/projects/js-toolkit/lerna.json

```diff
@@ -9,9 +9,7 @@
 		}
 	},
 	"npmClient": "yarn",
-	"packages": [
-		"packages/*"
-	],
+	"packages": ["packages/*"],
```

Comment:
Unrelated, so [already fixed separately here](https://github.com/liferay/liferay-frontend-projects/pull/294) along with [a proposal (ie. a PR) to stop it from happening again](https://github.com/liferay/liferay-frontend-projects/pull/295).

Reply:
:+1: